src/main.rs

```diff
@@ -916,7 +938,12 @@ fn main() -> anyhow::Result<()> {
         options.extension.strikethrough = true;
         options.parse.smart = true;
         options.render.unsafe_ = true;
-        let htmlified = markdown_to_html(&md_string, &options);
+
+        let mut plugins = comrak::ComrakPlugins::default();
+        let adapter = comrak::plugins::syntect::SyntectAdapter::new("InspiredGitHub");
```

Comment:
It would also be nice to be able to change this depending on the theme. It should be as simple as adding `pub code_highlighter: &'static str` to `Theme` and replacing `InspiredGithub` with `theme.code_highlighter`.

I think `InspiredGithub` looks really good for light theme, and I'm a fan of `base16-mocha.dark`. Please let me know if you have any other suggestions.

Reply:
That sounds good to me!

Eventually we can make it configurable so that users can pick any of the themes that `syntect` supports. `comrak` doesn't expose a way to set a theme other than the ones included by default AFAICT, but that still give users a handful of different options at least